problem/templates/problem/problem.html

```diff
@@ -117,4 +121,3 @@ <h4 class='modal-title'>Limits</h4>
   <br>
 </div>
 </body>

```

Comment:
For overall of this file your ' and " usage may be same.
e.g. 
\<div class='panel panel-default' style='float:right;width:47%'>
and
\<div class="tab-pane" id="tag">
Their quotation usages are different.